prog/probe.go

```diff
@@ -40,7 +40,10 @@ const (
 	defaultServiceHost = "https://cloud.weave.works:443"
 )
 
-var pluginAPIVersion = "1"
+var (
+	pluginAPIVersion = "1"
+	dockerEndpoint   = "unix:///var/run/docker.sock"
```

Comment:
Shouldn't this get `DOCKER_HOST` or something, to handle docker-machine / minikube cases?